README.md

```diff
@@ -299,7 +299,19 @@ Additionally, users can utilize the MiNiFi Toolkit Converter (version 0.0.1 - sc
 
     if you do not want to enable client certificate base authorization
     nifi.security.need.ClientAuth=false
-
+    
+### Configuring Volatile and NO-OP Repositories
+
+     in minifi.properties 
+     
+     # For Volatile Repositories:
+     nifi.flow.repository.class.name=VolatileRepository
+     nifi.provenance.repository.class.name=VolatileRepository
+     
+     # For NO-OP Repositories:
+	 nifi.flow.repository.class.name=NoOpRepository
```

Comment:
I think this works for how flowfile is currently implemented but if we have a model closer to that of the Java counterparts, not sure this makes a lot of sense outside of provenance.  For the other repositories we would not capture the current state of where a flowfile is in its overall processing nor have any content available between processor hops in the case of content.  Let me know your thoughts.